extensions/kind/src/templates/create-cluster-conf.mustache

```diff
@@ -3,6 +3,7 @@ apiVersion: kind.x-k8s.io/v1alpha4
 name: {{{ clusterName }}}
 nodes:
 - role: control-plane
+  {{{ extraConfig }}}
```

Comment:
it's like a pandora box if it's extra config here

it should probably only insert the image field if it's defined.